flutter/lib/src/navigation/sentry_navigator_observer.dart

```diff
@@ -84,6 +88,11 @@ class SentryNavigatorObserver extends RouteObserver<PageRoute<dynamic>> {
 
   ISentrySpan? _transaction;
 
+  static String? _currentRouteName;
+
+  /// Get the current route of the observer.
+  static String? get currentRouteName => _currentRouteName;
```

Comment:
I'm not personally a fan of static variables/methods. What do you think about adding a field in the options?
If it's not worth it, let's mark this as @internal